.codeclimate.yml

```diff
@@ -17,7 +17,7 @@ checks:
       threshold: 20
   method-lines:
     config:
-      threshold: 70
+      threshold: 100
```

Comment:
why do you need so many lines per method? if a method grows so big then it is an indication its doing a lot and needs to be broken down

Reply:
On it

Reply:
@Rodgers-M , with our current application design and setup, this is the component where all the different functions that affect creating an article are defined and later on passed as props, its a flow we had decided on rather than having the different submit functions in separate places, it would make it hard for someone new to this codebase. Also as it is, it allows us to easily plugin different features to the articles CRUD e.g this tag feature that is used when an article is being created.